text/0000-CNB-user-research-2021.md

```diff
@@ -0,0 +1,85 @@
+<ol>
+<li>How and where they find information today</li>
+<li>Which platforms they use CNBs on</li>
+<li>Expectations around inner loop development</li>
+</ol>
+<li>Anti-goals</li>
+<ol>
+<li>Who’s using what buildpacks brand</li>
+<li>Usability of the `pack` CLI, except as it relates to buildpack authorship</li>
+<li>Challenges of platform implementors integrating the lifecycle</li>
+<li>Challenges of platform operators doing governance</li> 
+</ol>
+</ol>
+
+# Process
+**Timeline:** 
+November 1, 2020 - February 28, 2021 (completed before Buildpacks Summit in the spring)
+ 
+**Checkpoints:**
+- [x]Create initial proposal for funding - Nov 2020
+- [x]Review existing documents - summary [doc](https://docs.google.com/document/d/1gEVCykAN2j4Ha6NzwHt1m5J1Jtdh16Riio1w3me7peY/edit#)  - Nov 2020
+- [x]Conduct SME interviews - Nov 2020
+- [x]Define research participants - screener survey [doc](https://docs.google.com/document/d/12PQsupIn3I50xC0AzC3ouLKY10V8KpbYh8Vnh8jiK6s/edit) - Dec 2020
+- [ ]Write script for first round - script [doc](https://docs.google.com/document/d/1rn7ojSXJNbYSzn8p4kaLNGFkT_5Q_zeD03cdZ5h-JlM/edit) - Dec 2020
+- [ ]Conduct 10-15 interviews with app developers about benefits and tradeoffs of buildpacks - Jan 2021
+- [ ]Review and synthesize interview findings -  findings [doc](https://docs.google.com/document/d/1anR_uma7ajr51xfvZAJsSSE5P2-bjPlL6EEBWz8pfTQ/edit#heading=h.7uxlulg7055g) - Jan + Feb 2021
+- [ ]Review findings with contributors and edit script - Jan 2021
+- [ ]Conduct 5-7 usability tests of simple buildpack authorship - list of authors [doc](https://docs.google.com/document/d/12PQsupIn3I50xC0AzC3ouLKY10V8KpbYh8Vnh8jiK6s/edit) - Feb 2021
+- [ ]Conduct survey - Feb 2021
+- [ ]Review and synthesize usability findings -  findings [doc](https://docs.google.com/document/d/1anR_uma7ajr51xfvZAJsSSE5P2-bjPlL6EEBWz8pfTQ/edit#heading=h.7uxlulg7055g) - Feb + Mar 2021
+- [ ]Present findings to contributors - Mar 2021
+- [ ]Conduct follow up usability tests with resulting authorship solutions - Apr 2021?
+
+
+# Participants
+- App developers currently using buildpacks 
+- App developers currently using dockerfiles or another container build technology
+  - A representative slice, not a specific subset 
+- Buildpack authors, or proxies, like:
+  - Senior Dev Ops folks / engineering leads who have a holistic sense of their project
+  - People maintaining a “blessed” base image to use with Dockerfiles
+  - Developer enablement teams (platform teams)
+
+
+# Collaborators
+- User Recruitment Partner: [gotomedia](https://www.gotomedia.com/)
+- Co-Researcher and Engineering Advisor: Natalie Arellano (VMware + CNB)
+- Supporting Researchers: Megan Taylor (VMware)
+- Supporting Interviewers and Notetakers: Sophie Wigmore (VMware + Paketo), Josh Zarrabi (VMware + Paketo), and Daniella Corricelli (VMware + Paketo)
+ 
+And always looking for more! Just leave a comment if you’d like to get involved :)
```

Comment:
I'd love to be a part! Happy to help with interviews, note-taking, etc.

Reply:
awesome! ill reach out, thank you